cms/plugin_rendering.py

```diff
@@ -264,6 +264,9 @@ def render_page_placeholder(self, slot, context, inherit, nodelist=None):
             nodelist=nodelist,
         )
 
+        if content:
+            return content
+
```

Comment:
I thought about it but decided against it because the next line is very different in logic.
Best to keep the lines separate and address the second line directly with the comment